iml-agent/src/daemon_plugins/postoffice.rs

```diff
@@ -52,36 +50,61 @@ fn start_route(mailbox: String) -> Trigger {
         let _ = fs::remove_file(&addr).await.map_err(|e| {
             tracing::debug!("Failed to remove file {}: {}", &addr, &e);
         });
-        let mut listener = UnixListener::bind(addr.clone()).map_err(|e| {
-            tracing::error!("Failed to open unix socket {}: {}", &addr, &e);
-            e
-        })?;
-        let mut incoming = listener.incoming().take_until(tripwire);
+
+        let conn = UnixStream::connect(&addr)
```

Comment:
Why is this changing?  This side should be the server listening for connections right?

Reply:
How many writers will there be on a socket at once?

Reply:
current usage between 0 and number of mounted targets.